net.pdl2ork.PdL2Ork.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.pdl2ork.PdL2Ork</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0+ and BSD-3-Clause</project_license>
```

Comment:
I don't think this will pass. It expects `AND` in uppercase.

Reply:
Also `GPL-3.0+` is deprecated and it should be `GPL-3.0-or-later`

(I don't think we fail on this yet)

Reply:
I will upload updated version of this file shortly.